targets/ar71xx-generic/profiles.mk

```diff
@@ -302,3 +302,8 @@ $(eval $(call GluonProfile,MR16,rssileds))
 $(eval $(call GluonProfileFactorySuffix,MR16))
 $(eval $(call GluonModel,MR16,mr16,meraki-mr16))
 $(eval $(call GluonModelAlias,MR16,meraki-mr16,meraki-mr66))
+
+# Carambola 2
```

Comment:
Please also add a comment with the vendor (like it's there for the other vendors)